### PR TITLE
negfix8 8.3 (new formula)

### PR DESCRIPTION
**Pull request by sbugert**
Bash script using imagemagick to convert scanned negatives to postive images.

### CODE REVIEW

File: Library/Formula/negfix8.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class Negfix8 < Formula
+  desc "Negfix8 turns scanned negative images into positives."
+  homepage "https://sites.google.com/site/negfix"
+  url "https://sites.google.com/site/negfix/downloads/negfix8.3.tgz"
+  sha256 "2f360b0dd16ca986fbaebf5873ee55044cae591546b573bb17797cbf569515bd"
+
+  depends_on "imagemagick" => %w(with-libtiff with-quantum-depth-16)
+
+  def install
+    bin.install 'negfix8'
```
Comment (DomT4) [inline Library/Formula/negfix8.rb]: This should be double quotes.

File: Library/Formula/negfix8.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  desc "Negfix8 turns scanned negative images into positives."
+  homepage "https://sites.google.com/site/negfix"
+  url "https://sites.google.com/site/negfix/downloads/negfix8.3.tgz"
+  sha256 "2f360b0dd16ca986fbaebf5873ee55044cae591546b573bb17797cbf569515bd"
+
+  depends_on "imagemagick" => %w(with-libtiff with-quantum-depth-16)
+
+  def install
+    bin.install "negfix8"
+  end
```
Comment (bfontaine) [inline Library/Formula/negfix8.rb]: Could you add a test? Thanks!

**Comment by sbugert** [comment]
Is there a way to add a test image file or will running it without user input suffice (what is considered a "bad test" in the cookbook)?

**Comment by mikemcquaid** [comment]
> Is there a way to add a test image file or will running it without user input suffice (what is considered a "bad test" in the cookbook)?

Yeh, check `imagemagick.rb` for an example.

File: Library/Formula/negfix8.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+class Negfix8 < Formula
+  desc "Negfix8 turns scanned negative images into positives."
+  homepage "https://sites.google.com/site/negfix"
+  url "https://sites.google.com/site/negfix/downloads/negfix8.3.tgz"
+  sha256 "2f360b0dd16ca986fbaebf5873ee55044cae591546b573bb17797cbf569515bd"
+
+  depends_on "imagemagick" => %w(with-libtiff with-quantum-depth-16)
```
Comment (tdsmith) [inline Library/Formula/negfix8.rb]: `with-libtiff` is the default now (as of today) so it should be removed here. You can just use `depends_on "imagemagick" => "with-quantum-depth-16"`.

**Comment by tdsmith** [comment]
Great test, thanks!

**Pull request closed without merging**